lib/fxpay/index.js

```diff
@@ -39,6 +34,7 @@
 
 
   exports.validateAppReceipt = function validateAppReceipt(callback) {
+    settings.initialize();
```

Comment:
Maybe we should move forward with this as is for now as this is blocking the webfighter demo.

As another idea I wonder if we should have it so that the end-user instantiates the fxpay object. The constructor could init the adapter and and take the settings object. This might end up with a more obvious usage pattern. Not sure how big the impact of that change might be.